systemtests/src/test/java/io/enmasse/systemtest/iot/registry/DeviceRegistryTestBase.java

```diff
@@ -4,43 +4,75 @@
  */
 package io.enmasse.systemtest.iot.registry;
 
-import io.enmasse.systemtest.CustomLogger;
-import io.enmasse.systemtest.Endpoint;
-import io.enmasse.systemtest.bases.IoTTestBaseWithShared;
-import io.enmasse.systemtest.iot.CredentialsRegistryClient;
-import io.enmasse.systemtest.iot.DeviceRegistryClient;
-import io.enmasse.systemtest.iot.HttpAdapterClient;
-import io.vertx.core.json.JsonArray;
-import io.vertx.core.json.JsonObject;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Tag;
-import org.junit.jupiter.api.Test;
-import org.slf4j.Logger;
-
-import java.net.HttpURLConnection;
-import java.time.Instant;
-import java.util.Map;
-
 import static io.enmasse.systemtest.TestTag.sharedIot;
+import static io.enmasse.systemtest.TestTag.smoke;
 import static io.enmasse.systemtest.apiclients.Predicates.in;
 import static java.net.HttpURLConnection.HTTP_ACCEPTED;
 import static java.net.HttpURLConnection.HTTP_UNAUTHORIZED;
 import static java.net.HttpURLConnection.HTTP_UNAVAILABLE;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
+import java.net.HttpURLConnection;
+import java.time.Instant;
+import java.util.Map;
+import java.util.UUID;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.slf4j.Logger;
+
+import io.enmasse.iot.model.v1.IoTConfig;
+import io.enmasse.iot.model.v1.IoTProject;
+import io.enmasse.systemtest.CustomLogger;
+import io.enmasse.systemtest.Endpoint;
+import io.enmasse.systemtest.bases.IoTTestBase;
+import io.enmasse.systemtest.iot.CredentialsRegistryClient;
+import io.enmasse.systemtest.iot.DeviceRegistryClient;
+import io.enmasse.systemtest.iot.HttpAdapterClient;
+import io.enmasse.systemtest.utils.IoTUtils;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
 @Tag(sharedIot)
-class DeviceRegistryTest extends IoTTestBaseWithShared {
+@Tag(smoke)
+public abstract class DeviceRegistryTestBase extends IoTTestBase {
 
-    private Logger log = CustomLogger.getLogger();
+    protected Logger log = CustomLogger.getLogger();
```

Comment:
I don't understand why it is protected?

Reply:
my fault I must had change it and have forget about it

Reply:
ok :)